packages/grid/_modules_/grid/components/GridRow.tsx

```diff
@@ -61,13 +63,20 @@ export const GridRow: React.FC<GridRowProps> = ({
     minHeight: rowHeight,
   };
 
+  const rowClassName =
+    isFunction(options.getRowClassName) &&
```

Comment:
Curious helper, I don't understand how it's helpful, I would have personally done:

```suggestion
    typeof options.getRowClassName === 'function' &&
```

Reply:
- better readability
- avoid to think/reinvent the wheel all the time.

Reply:
I personally disagree with both points. isFunction reinvents the wheel, what type of already does and is less clear than typeof as it's yet another module we have to learn to understand what it does. But at the end of the day, I think that it's matter of past learnings and taste, so no objections to have it.